app/src/org/commcare/android/view/GridEntityView.java

```diff
@@ -361,12 +366,13 @@ private View getView(Context context, String multimediaType, String horzAlign, S
                         break;
                 }
                 retVal.setPadding(CELL_PADDING_HORIZONTAL, CELL_PADDING_VERTICAL, CELL_PADDING_HORIZONTAL, CELL_PADDING_VERTICAL);
-                // image loading is handled asyncronously by the TCImageLoader class to allow smooth scrolling
                 if (rowData != null && !rowData.equals("")) {
                     if (mImageLoader != null) {
-                        mImageLoader.display(rowData, ((ImageView) retVal), R.drawable.info_bubble);
+                        mImageLoader.display(rowData, ((ImageView) retVal), R.drawable.info_bubble,
```

Comment:
I don't think we wanna scale this, if possible. The drawable bitmaps are super efficient at getting themselves in/out of memory, and this'll result in there being a new copy of it